api/src/modules/indicator-records/indicator-records.service.ts

```diff
@@ -156,69 +154,70 @@ export class IndicatorRecordsService extends AppBaseService<
    */
   // TODO still not adapted to modular indicators, because the performance gets hit drastically on the source import. Pending to be worked on
   async createIndicatorRecordsForAllSourcingRecords(): Promise<void> {
+    const indicatorMap: Record<string, any> = await this.getIndicatorMap();
+    // Using a hardcoded list of indicator types because this function is still based on
+    // a calculation query with hardcoded indicators,
+    const indicators: INDICATOR_TYPES[] = [
```

Comment:
Maybe something like 
```
Object.keys(INDICATOR_TYPES)
```
?
However, shouldn't we perform calculations for indicators that are only available in the DB? Or is this handled at some other point?